hypothesis-python/tests/codemods/test_codemods.py

```diff
@@ -64,13 +64,13 @@ def test_substitution(self) -> None:
         before = """
             import hypothesis.strategies as st
 
-            st.floats(0, 1, False, False, 32)
+            st.floats(0, 1, False, False, True, 32)
             st.fractions(0, 1, 9)
         """
         after = """
             import hypothesis.strategies as st
 
-            st.floats(0, 1, allow_nan=False, allow_infinity=False, width=32)
+            st.floats(0, 1, allow_nan=False, allow_infinity=False, allow_subnormal=True, width=32)
```

Comment:
We're going to fix this by hard-coding the old signature of `floats()` in `hypothesis.extra.codemods` - otherwise if you actually had the previous code the codemod would output `allow_subnormal=32` and... that wouldn't work so well 😉